src/app/components/Pagination/index.tsx

```diff
@@ -0,0 +1,203 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import { useContext } from 'react';
+import {
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+  GEL_GROUP_2_SCREEN_WIDTH_MAX,
+  GEL_GROUP_3_SCREEN_WIDTH_MAX,
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+  GEL_GROUP_4_SCREEN_WIDTH_MIN,
+} from '#app/legacy/psammead/gel-foundations/src/breakpoints';
```

Comment:
You may be able to use the ThemeProvider media query values from: https://github.com/bbc/simorgh/blob/729de2d6ab0b0244d0e567433430a2bf476b70a0/src/app/components/ThemeProvider/mediaQueries.ts#L7-L8 instead of Psammead. They _should_ provide the same values.